nemo_text_processing/text_normalization/en/data/electronic/domain.tsv

```diff
@@ -1,11 +1,12 @@
 .com	dot com
 .org	dot org
 .gov	dot gov
-.uk	dot uk
-.fr	dot fr
+.uk	dot UK
+.fr	dot FR
 .net	dot net
-.br	dot br
-.in	dot in
-.ru	dot ru
-.de	dot de
-.it	dot it
\ No newline at end of file
+.br	dot BR
+.in	dot IN
+.ru	dot RU
+.de	dot DE
+.it	dot IT
+.jpg	dot J peg
```

Comment:
why `j peg` but `gmail`?

Reply:
oh because I did not treat server in particular (too many). I only treat domain